modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileIntegrationTest.java

```diff
@@ -1277,6 +1321,38 @@ private void _assertSamlSsoRequestContexts(
 			samlSsoRequestContexts.size());
 	}
 
+	private String _buildEncodedSamlRequest() throws Exception {
+		AssertionConsumerService assertionConsumerService =
```

Comment:
HEy @jgarcialr, just inline the `assertionConsumerService` and the `singleSignOnService`. After this change we can forward it to BChan

Reply:
Oh, I forgot those, done